src/components/RelatedVideos/RelatedVideo.test.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import RelatedVideo from './RelatedVideo.component';
+import { AppProvider } from '../../utils/AppContext.provider';
+
+describe('Testing RelatedVideo component', () => {
+  let wrapper;
+  const videoData = [
+    {
+      id: {
+        kind: 'youtube#video',
+        videoId: 'nmXMgqjQzls',
+      },
+      snippet: {
+        title: 'This is a title 1',
+        description: 'this is a desciption1',
+        thumbnails: {
+          high: {
+            url: 'https://yt3.ggpht.com/ytc/AAUvwnighSReQlmHl_S_vSfvnWBAG5Cw4A0YxtE0tm5OpQ=s800-c-k-c0xffffffff-no-rj-mo',
+          },
+        },
+      },
+    },
+    {
+      id: {
+        kind: 'youtube#video',
+        videoId: 'fgfhhfgh',
+      },
+      snippet: {
+        title: 'This is a title 2',
+        description: 'this is a desciption 2',
+        thumbnails: {
+          high: {
+            url: 'https://yt3.ggpht.com/ytc/AAUvwnighSReQlmHl_S_vSfvnWBAG5Cw4A0YxtE0tm5OpQ=s800-c-k-c0xffffffff-no-rj-mo',
+          },
+        },
+      },
+    },
+  ];
+  beforeEach(() => {
+    wrapper = render(
+      <AppProvider>
+        <BrowserRouter>
+          <RelatedVideo
+            currentVideoId="nmXMgqjQzls"
+            videos={{ items: videoData }}
+            path="video"
+          />
+        </BrowserRouter>
+      </AppProvider>
+    );
```

Comment:
```js
wrapper = render(
      <AppProvider>
        <BrowserRouter>
          <RelatedVideo
            currentVideoId="nmXMgqjQzls"
            videos={{ items: videoData }}
            path="video"
          />
        </BrowserRouter>
      </AppProvider>
    );
```

If you are duplicating this code (the Providers and Router wrappers) in other tests, try to figure it out how to encapsulate or abstract that logic and reuse it